### DISCUSSION TITLE
Stellarium not compatible with Nuc

### DISCUSSION THREAD

**Discussion by Denodan** [Stellarium/stellarium#2688] [Q&A]
![20220923_223342](https://user-images.githubusercontent.com/114269817/192073899-1c2d273a-54a5-45b8-8960-9473145a9691.jpg)

Why do I get this? Have all the latest drivers, just won't work with my Nuc i7 7th gen on Windows 11

**Comment by Denodan** [comment]
Tried version 20.2 , works, later versions do not

**Reply by gzotti** [reply]
https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version#windows-10

I have a 4th-gen NUC with Ubuntu. Works. We conclude this is a driver problem out of our hands. Once more, try ANGLE mode.

**Reply by Denodan** [reply]
No the angle mode does not work. Any later than 20.2 won't work

**Reply by Denodan** [reply]
But are you guys running it on windows 11? That's the question? And may well not beca driver issue, but a windows 11 issue?

**Reply by 10110111** [reply]
Did you try the driver version suggested in the FAQ? The screenshot looks like a problem in the driver, not in Stellarium. (I don't have NUC myself, so can't test.)

**Reply by Denodan** [reply]
It won't allow me to install the latest driver

**Reply by gzotti** [reply]
No Win11 on a 4th-gen (?; 2015) NUC... Still, out of our hands.
